### DISCUSSION TITLE
EA0326GMP刷uboot之后无法ssh连接

### DISCUSSION THREAD

**Discussion by Tian-Jionglu** [immortalwrt/immortalwrt#1223] [Q&A] [answered]
### Describe the bug

EA0326GMP刷uboot后，192.168.1.1可以ping通，但是无法通过ssh连接。

```
root@cmcc:/tmp/imm# mtd write preloader.bin BL2
Could not open mtd device: BL2
Can't open device for writing!
root@cmcc:/tmp/imm# mtd write uboot.fip FIP
Unlocking FIP ...

Writing from uboot.fip to FIP ...
````

### ImmortalWrt version

23.05 SNAPSHOT

### ImmortalWrt release

23.05 SNAPSHOT

### ImmortalWrt target/subtarget

MTK7981BA

### Device

EA0326GMP

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by Tian-Jionglu** [comment, marked as answer]
第一个问题已经解决，是tftpd没有正常使用，参考问题：https://github.com/immortalwrt/immortalwrt/discussions/1008#discussioncomment-8355405
碰到第二个ssh内sysupgrade升级时报错：
```
root@ImmortalWrt:/tmp/image# sysupgrade *.itb
Cannot save config while running from ramdisk.
Thu Jan  1 00:02:30 UTC 1970 upgrade: Commencing upgrade. Closing all shell sessions.
Command failed: Connection failed
root@ImmortalWrt:/tmp/image# Connection to 192.168.1.1 closed by remote host.
Connection to 192.168.1.1 closed.

**Reply by Tian-Jionglu** [reply]
尝试使用以下命令，都无法刷新成功
1. `sysupgrade *.itb`
2. `sysupgrade -F -n *.itb`

**Comment by Tian-Jionglu** [comment]
tftpd的地址必须是`192.168.1.254`

**Reply by puffinjiang** [reply]
刷入23.05.03版本的uboot时候遇到了同样的问题，并且把本机的ip 也改成了192.168.1.254，ssh依然连接不上，请问还需要有别的配置吗？

**Reply by Tian-Jionglu** [reply]
确认下你的状态：
1. 已经刷uboot；
2. 重启机器；
3. 可以ping通192.168.1.1；
4. 无法通过tftpd启动recovery内核；

是这样吗？
